src/main/scala/xiangshan/mem/vector/VSegmentUnit.scala

```diff
@@ -347,10 +347,8 @@ class VSegmentUnit (implicit p: Parameters) extends VLSUModule
     )
   }.elsewhen(state === s_fof_fix_vl){ // writeback uop
     stateNext := Mux(!fofBufferValid, s_idle, s_fof_fix_vl)
-
-  }.otherwise{
+  }.otherwise{ // unknown state
     stateNext := s_idle
-    XSError(true.B, s"Unknown state!\n")
```

Comment:
If it is really difficult to new XSError to do this, use `assert(false.B)` instead of simply delete this line.

Reply:
Sure. I will make a change